### DISCUSSION TITLE
I was billed 3 times instead of 1 time for a one-time sponsor.

### DISCUSSION THREAD

**Discussion by animegers** [community/community#147731] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

On 19 December i sponsored a Friend with one-time sponsor which was 12 dollars (1500 BDT). GitHub took the payment instantly but it was supposed to be only one-time payment with 12 dollars but they took it 3 times making it 36 dollars! Am i missing something or what's going on? I also opened a support ticket 3 days ago but they are not responding....
[github-animegers-receipt-2024-12-19.pdf](https://github.com/user-attachments/files/18221538/github-animegers-receipt-2024-12-19.pdf)
[bank Statement.pdf](https://github.com/user-attachments/files/18221540/bank.Statement.pdf)

Also here's my all time payment history... I had only one payment which was that 12$ payment but the other 2 payment also went to GitHub's us terminal.

![IMG_20241222_161603](https://github.com/user-attachments/assets/f783fd67-2a65-4c44-b2fc-bf12ad7b8135)

**Comment by mecodeatlas** [comment]
Hi @animegers!

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
